### DISCUSSION TITLE
More Fonts Families Out of the Box

### DISCUSSION THREAD

**Discussion by sirinath** [tailwindlabs/tailwindcss#1424] [General]
Instead of having to add custom fonts is it possible to add more of the major free web-based fonts hosted on Google Fonts.

**Comment by zaydek** [comment]
Adding a small feature like this would increase the load for all prospective developers and users. As I understand it, Tailwind’s default font preferences are exclusively based on fonts that ship on operating systems (so there’s no additional cost for prospective users).

That being said, I have a snippet of code (just a few lines long) that I copy and paste between projects. I’m sure you could also publish code as a plugin so you could consume it more easily, but that requires more maintenance, etc.

**Comment by sirinath** [comment]
Maybe this can great if there is a way to add all say fonts by something like `@tailwind googlefonts`, `@tailwind fontawesome5` and simply use them, especially if you are just starting to use tailwin.

**Comment by tlgreg** [comment]
Feels like that's a bit out of scope for `tailwindcss`.
There are tools that can do a similar job though.
https://github.com/jonathantneal/postcss-font-magician, for example, injects the `@font-face` rules based on `font-family` usage. If it's after `tailwindcss` in the `postcss` plugin list you can just add a font to your `fontFamily` config and it will work together.
